### PR TITLE
Allow CI builds not running as root

### PR DESCRIPTION
**Pull request by emanuelez**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,8 +40,14 @@ RUN echo y | android update sdk --no-ui --all --filter extra-android-m2repositor
...
+    cd /opt/android-ndk-tmp && \
+    wget -q http://dl.google.com/android/ndk/android-ndk-r10e-linux-x86_64.bin -O android-ndk.bin && \
+    chmod a+x ./android-ndk.bin && \
+    ./android-ndk.bin && \
+    mv android-ndk-r10e /opt/android-ndk && \
+    rm -rf /opt/android-ndk-tmp && \
+    chmod -R a+rX /opt/android-ndk
+
+# Make the SDK universally readable
+RUN chmod -R a+rX /opt/android-sdk-linux
```
Comment (beeender) [inline Dockerfile]: `android-sdk-linux` or `android-sdk` ?

**Comment by emanuelez** [comment]
@alebsack @cmelchior @teotwaki

File: Dockerfile (hunk 1 above)
Comment (emanuelez) [inline Dockerfile]: it's `android-sdk-linux` nowadays. https://github.com/realm/realm-java/pull/3335/commits/a531df4859414adddfee4e6f9099d67cf0def10e

**Comment by teotwaki** [comment]
:+1:

**Pull request merged**